pilot/pkg/networking/plugin/authz/rbac.go

```diff
@@ -366,7 +449,9 @@ func permissionForKeyValues(key string, values []string) *policyproto.Permission
 			}, nil
 		}
 	default:
-		log.Errorf("ignored unsupported constraint key: %s", key)
+		if !shouldMatchAttribute(key) {
```

Comment:
it's a bit strange to call this function here just for logging.

Reply:
Updated the function name and added a comment.